install.sh

```diff
@@ -0,0 +1,29 @@
+# Test for root
+if [ "$EUID" -ne 0 ]
+  then echo "Please run as root (sudo)"
+  exit
+fi
+
+# Copying the agent to its final destination
+cp -r jumper-logging-agent /opt
+cd /opt/jumper-logging-agent
+
+# Install pip and virtualenv
+apt-get install -y python-pip
```

Comment:
`apt-get install` in the install script might have surprising side effects. 
- It can install pending security updates, which the user did not intend to.
- If there was no `apt-get update` recently, it might install an old version
- If there was no `apt-get update` at all (e.g, on a newly installed system), it might not find the package at all.

I think it's preferable to check for the existence of pip and if not exists, exit with an error suggesting to run `apt-get install python-pip`